architecture.yml

```diff
@@ -433,3 +433,16 @@ architecture:
         image: datascience-notebook
         tag: 4.2.6
         pipeline_dependent: true
+  cmis-notebook:
+    latest:
+      parent:
+        owner: ucphhpc
+        image: python-notebook
```

Comment:
As discussed with @SebastianPrehn on chat, since the `cmis-notebook` is going to use a non-default python version 3.9 from what I can gather from the working_dockerimage reference (compared to our current python-notebook which defines it to be 3.12) I would recommend that it instead inherits from the underlying `base-notebook` instead, which ensures that it wont conflict with the existing environment

Reply:
Thank you @rasmunk.
I have tried this already, and it makes no difference. I get the same behavior.